app/javascript/images/logo_white.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="125" height="30" viewBox="0 0 125 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
**suggestion (non-blocking)**: svgs can be compressed a lot in a tool like https://jakearchibald.github.io/svgomg/ while still maintaining the exact same appearance. For example this one can have 60% less.